ArduCopter/Parameters.cpp

```diff
@@ -991,6 +991,7 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @Description: Bitmask of additional options for battery, radio, & GCS failsafes. 0 (default) disables all options.
     // @Values: 0:Disabled, 1:Continue if in Auto on RC failsafe only, 2:Continue if in Auto on GCS failsafe only, 3:Continue if in Auto on RC and/or GCS failsafe, 4:Continue if in Guided on RC failsafe only, 8:Continue if landing on any failsafe, 16:Continue if in pilot controlled modes on GCS failsafe, 19:Continue if in Auto on RC and/or GCS failsafe and continue if in pilot controlled modes on GCS failsafe
     // @Bitmask: 0:Continue if in Auto on RC failsafe, 1:Continue if in Auto on GCS failsafe, 2:Continue if in Guided on RC failsafe, 3:Continue if landing on any failsafe, 4:Continue if in pilot controlled modes on GCS failsafe, 5:Release Gripper
+    // @Values: 0:None, 1:Continue if in Auto on RC failsafe, 2:Continue if in Auto on GCS failsafe, 4:Continue if in Guided on RC failsafe, 8:Continue if landing on any failsafe, 166:Continue if in pilot controlled modes on GCS failsafe, 32:Release Gripper
```

Comment:
This is listed above it except for the "32:Release Gripper" part

Reply:
oops...meant to just add the last one...nice catch..thanks